src/main/java/com/google/jspecify/nullness/NullSpecVisitor.java

```diff
@@ -115,10 +116,11 @@ protected String extraArgForReturnTypeError(Tree tree) {
      * the possibly null value (possibly both!). However, this gets tricky: If the branches return
      * `Foo?` and `Foo*`, then we ideally want to emphasize the `Foo?` branch *but*, at least in
      * "strict mode," not altogether ignore the `Foo*` branch.
-     */
+     *
```

Comment:
```suggestion
      /
```

Maybe leave the slash to mirror what you did on line 106.